packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -624,6 +625,7 @@ For a request to be a _bounded_ request it must meet the following request param
   - `until` must be specified.
 
 Bounded requests to the `/api/v1/logs` API have the following semantics:
+  - They are time-filtered by their `published` field (unlike [Polling Requests](#polling-requests)).
```

Comment:
Nit: consider "event `published`" to differentiate it from possible confusion with a request query parameter.